docs/sources/enterprise/access-control/fine-grained-access-control-references.md

```diff
@@ -30,8 +31,8 @@ The reference information that follows complements conceptual information about
-| Editor         | `fixed:datasource:editor:read`                                                                                                                                                                                                                                                                                                                                                                |
+| Built-in role | Associated role                                                                                                                                                                                                                                                                                                                                                                                                            | Description                                                                                                                                                |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Grafana administrator  | `fixed:permissions:admin:edit`<br>`fixed:permissions:admin:read`<br>`fixed:provisioning:admin`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`<br>`fixed:users:admin:edit`<br>`fixed:users:admin:read`<br>`fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:ldap:admin:edit`<br>`fixed:ldap:admin:read`<br>`fixed:server:admin:read`<br>`fixed:settings:admin:read`<br>`fixed:settings:admin:edit` | Allow access to the same resources and permissions that the [Grafana server administrator]({{< relref "../../permissions/_index.md#grafana-server-admin-role" >}}) has by default. |
+| Administrator          | `fixed:users:org:edit`<br>`fixed:users:org:read`<br>`fixed:reporting:admin:edit`<br>`fixed:reporting:admin:read`                                                                                                                                                                                                                                                                                                            | Allow access to the same resources and permissions that the [Organization administrator]({{< relref "../../permissions/organization_roles.md" >}}) has by default.                               |
```

Comment:
They're assigned to the strings `"Admin"` and `"Grafana Admin"` respectively. Do we have anything that makes it clear that that's when you want to change the organizational administrator's permissions, you should provision for `Admin` rather than `Administrator` or should that too be introduced? (not in this PR, regardless. We shouldn't let PRs be blocked by work that's not directly related to the PR)

Reply:
The [provisioning docs](https://grafana.com/docs/grafana/latest/enterprise/access-control/provisioning/#built-in-role-assignments) and [API docs](https://grafana.com/docs/grafana/latest/http_api/access_control/#create-a-built-in-role-assignment) state this clearly. 

In other places it is referencing to the current docs where it's not very clear in either way. I'm not against this change (so I accepted the suggestion), however I agree that it won't make things less confusing for users. So if this is a one step forward towards consistent glossary and eliminates future work, I'm fine to update. But @osg-grafana would appreciate your call to unblock this PR, we can take this conversation to private and understand how we can move forward with FGAC docs.

Reply:
That's good enough for me :+1: Thanks for the links